src/data/content/design-system/componenti/breadcrumbs.yaml

```diff
@@ -302,7 +304,7 @@ tabs:
             title: Specifiche di design
             headingLevel: 2
             text: |
-              *In stesura*
+              ![Specifiche di design dei breadcrumbs](/images/design-system/componenti/breadcrumbs-specs.png)
```

Comment:
Nell'immagine delle spec mancano i riferimenti delle spaziature.

Ho provato a rigenerarle sull'elemento singolo, li trovi su Figma (vedi allegati):
![breadcrumb-spec-2](https://github.com/italia/designers.italia.it/assets/69706/bb6a2da3-c8f2-4e55-9b14-df6d95f1a8b2)
![breadcrumb-spec](https://github.com/italia/designers.italia.it/assets/69706/b596159a-32cd-435d-8efd-60775c31a0be)

Manca però un modo per fare vedere lo spazio tra items, il plugin non lo genera...

Reply:
Ok grazie! Edito a mano e ripubblico.